tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/display.py

```diff
@@ -9,7 +9,7 @@
 
 class Display(Base):
 
-    _page_locator = (By.ID, 'sound')
```

Comment:
Oops, Pycharm tricked me then. It showed me that it was added in bug 909718, when I loaded the git log for that single line. Sorry about that